src/docs/asciidoc/reference/configuration/configuration-properties.adoc

```diff
@@ -1,6 +1,26 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
+ZetaPush provides the `application.json` file to let you configure the behavior of your application.
 
+For now, the `application.json` file is mainly used for the <<reference#integrate_user_management, Standard User Workflow>> but you can also <<reference#configuration_properties ,configure your own properties for your application>>.
+The list of available properties for the <<reference#integrate_user_management, Standard User Workflow>> are listed in <<reference#customize_standard_user_workflow, this section>>.
+
+.New properties will added
+[NOTE]
+====
+New _Cloud Services_ will be developed (to handle data, security, etc...).
+With them, some available properties of the `application.json` will be added.
```

Comment:
some ~available~ properties ~of the application.json will~ **may** be added **and configurable through the application.json file**